apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingDao.kt

```diff
@@ -129,8 +135,37 @@ class BehandlingDao(private val connection: () -> Connection) {
                 objectMapper.readValue(it)
             },
             prosesstype = rs.getString("prosesstype").let { Prosesstype.valueOf(it) },
-            kilde = rs.getString("kilde").let { Vedtaksloesning.valueOf(it) }
+            kilde = rs.getString("kilde").let { Vedtaksloesning.valueOf(it) },
+            revurderingInfo = revurderingInfo
         )
+    }
+
+    fun lagreRevurderingInfo(id: UUID, revurderingInfo: RevurderingInfo, kilde: Grunnlagsopplysning.Kilde) {
+        connection().prepareStatement(
+            """
+                INSERT INTO revurdering_info(behandling_id, info, kilde)
+                VALUES(?, ?, ?) ON CONFLICT(behandling_id) DO UPDATE SET info = excluded.info, kilde = excluded.kilde
```

Comment:
Det er meningen å kunne overskride revuderingsinfo? Er det fordi vi aldri sletter tidligere info?

Reply:
Det er når man behandler saken. Litt som å lagre en behandling -- men det er skrevet som en upsert heller enn å sette inn null for alle nye revurderinger